### ISSUE TITLE
54.1 Join (dependencies)

### ISSUE THREAD

**Issue by mgeisler** [google/comprehensive-rust#1330]
### Discussed in https://github.com/google/comprehensive-rust/discussions/1317

<div type='discussions-op-text'>

<sup>Originally posted by **njr0** October  6, 2023</sup>
I had to add these dependencies to may `Cargo.toml` to make it work. As with the last few suggestions, I think it will help to includes these in the speaker notes or main text, especially since this can't be run in the playground. (And who wants to be in a playground anyway?)

```
[dependencies]
tokio = { version = "1.32.0", features = ["full"] }
reqwest = { version = "0.11.22", features = ["blocking", "rustls-tls"] }
futures = "0.3"
anyhow = "1.0"
```
</div>

**Comment by mgeisler** [comment]
> Is this the section you are referring to [src/async/control-flow/join.md](https://github.com/google/comprehensive-rust/blob/main/src/async/control-flow/join.md) ?

Yeah, that sounds right, @njr0 can confirm and give more input here.
 
> I wouldn't mind working on this.

Awesome, thanks a bunch!

**Comment by mgeisler** [comment]
In general, @njr0 had a ton of excellent feedback and I probably only captured some of it in the issues. It's all available [in the discussions](https://github.com/google/comprehensive-rust/discussions?discussions_q=is%3Aopen+author%3Anjr0+).

**Comment by djmitche** [comment]
There are some instructions for running the course here:

https://github.com/google/comprehensive-rust/blob/main/src/running-the-course/course-structure.md#concurrency-in-rust

Perhaps those should be linked in some/all of the speaker notes for the concurrency section?

**Comment by 0scvr** [comment]
> There are some instructions for running the course here:
> 
> https://github.com/google/comprehensive-rust/blob/main/src/running-the-course/course-structure.md#concurrency-in-rust
> 
> Perhaps those should be linked in some/all of the speaker notes for the concurrency section?

In that case, I think https://github.com/google/comprehensive-rust/blob/77c55ef10ea786f7f4e66c7803f2a7304fabc967/src/running-the-course/course-structure.md?plain=1#L57-L62 is incomplete. Tokio isn't the only crate used. e.g. the `futures`, `anyhow` and `reqwest` crates are used here https://google.github.io/comprehensive-rust/async/control-flow/join.html but aren't
